### PR TITLE
(BKR-326) Fixes FreeBSD install to use ports

### PR DESCRIPTION
**Pull request by petems**

### CODE REVIEW

**Comment by anodelman** [comment]
retest this please

File: lib/beaker/dsl/install_utils/foss_utils.rb [hunk 1]
```diff
@@ -560,6 +562,27 @@ def install_a_puppet_msi_on(host, opts)
...
 
+        # Installs Puppet and dependencies from FreeBSD ports
+        #
+        # @param [Host] host The host to install packages on
+        # @param [Hash{Symbol=>String}] opts An options hash
+        # @option opts [String] :version The version of Puppet to install (shows warning)
+        #
+        # @return nil
+        # @api private
+        def install_puppet_from_freebsd_ports_on( host, opts )
```
Comment (anodelman) [inline lib/beaker/dsl/install_utils/foss_utils.rb]: Using the _on suffix in the host name means that you should be able to pass a single host or an array of hosts - please use the block_on method to cycle through those.

Comment (petems) [inline lib/beaker/dsl/install_utils/foss_utils.rb]: Does adding the alias method fix that?

Comment (anodelman) [inline lib/beaker/dsl/install_utils/foss_utils.rb]: No, as this is an _on method that doesn't work correctly with an array of hosts.

**Comment by anodelman** [comment]
Last review comment still current.

**Comment by anodelman** [comment]
Will close out next week due to inactivity.

File: lib/beaker/dsl/install_utils/foss_utils.rb (hunk 1 above)
Comment (petems) [inline lib/beaker/dsl/install_utils/foss_utils.rb]: But none of the other `_on` methods have that ability? This method has the same implementation as `install_puppet_from_deb_on` which gets the host given to it from the `block_on` block on https://github.com/petems/beaker/blob/BKR-326-fix_freebsd_install/lib/beaker/dsl/install_utils/foss_utils.rb#L231

Am I missunderstanding something? I can change the method, but it seems to be the same logic as the rest of the methods.

**Comment by anodelman** [comment]
Closing due to inactivity.

**Pull request closed without merging**